_config.yml

```diff
@@ -1,6 +0,0 @@
-baseurl: "/~wics"
-sitename: "UM WICS"
-locale: "en-US"
-
-collections:
-  - co-op-companies
```

Comment:
Looks like this file might have been deleted by accident?

Reply:
Yeah sorry about that. We thought we created this file by accident when adding a modal, but it turns out we dragged it from somewhere into the assets folder and deleted it while not knowing it was an actual file of the website.

Reply:
Okay I restored the file in my recent commit.

Reply:
No worries, it happens. Thanks for making the changes!